src/main/java/com/gooddata/executeafm/afm/SimpleMeasureDefinition.java

```diff
@@ -98,9 +125,44 @@ public String getUri() {
         return getItem().getUri();
     }
 
+    /**
+     * Returns the qualifier used by the {@link com.gooddata.md.Metric}.
+     *
+     * @return qualifier used by the metric, which is its {@link #item}.
+     *
+     * @throws IllegalStateException
+     *         The exception is thrown when {@link #item} has not been set.
+     * @deprecated Use {@link #getObjQualifiers()} instead.
+     */
     @Override
+    @Deprecated
     public ObjQualifier getObjQualifier() {
-        return getItem();
+        return getObjQualifiers().stream()
+                .findFirst()
+                .orElseThrow(() -> new IllegalStateException("The `item` of the object has not been set!"));
```

Comment:
not sure about backticks - we don't use them anywhere as far as I know

Reply:
Removed by revert of the method.